securedrop_client/locale/en_XA/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,372 @@
+msgstr[1] "[!!!Retrieving {message_count} messages!!!]"
+
+msgid "The SecureDrop server cannot be reached. Trying to reconnect..."
+msgstr "[!!!!!!The SecureDrop server cannot be reached. Trying to reconnect...!!!!!!]"
+
+msgid ""
+"Could not reach the SecureDrop server. Please check your \n"
+"Internet and Tor connection and try again."
+msgstr ""
+"[!!!!!!!!!!Could not reach the SecureDrop server. Please check your \n"
+"Internet and Tor connection and try again.!!!!!!!!!!]"
+
+msgid ""
+"Those credentials didn't work. Please try again, and \n"
+"make sure to use a new two-factor code."
+msgstr ""
+"[!!!!!!!!!Those credentials didn't work. Please try again, and \n"
+"make sure to use a new two-factor code.!!!!!!!!!]"
+
+msgid "That didn't work. Please check everything and try again."
+msgstr "[!!!!!That didn't work. Please check everything and try again.!!!!!]"
+
+msgid "You must sign in to perform this action."
+msgstr "[!!!!You must sign in to perform this action.!!!!]"
+
+msgid "Your session expired. Please log in again."
+msgstr "[!!!!Your session expired. Please log in again.!!!!]"
+
+msgid "Failed to update star."
+msgstr "[!!Failed to update star.!!]"
+
+msgid "File does not exist in the data directory. Please try re-downloading."
+msgstr "[!!!!!!File does not exist in the data directory. Please try re-downloading.!!!!!!]"
+
+msgid "The file download failed. Please try again."
+msgstr "[!!!!The file download failed. Please try again.!!!!]"
+
+msgid "Failed to delete conversation at server"
+msgstr "[!!!Failed to delete conversation at server!!!]"
+
+msgid "Failed to delete source at server"
+msgstr "[!!!Failed to delete source at server!!!]"
+
+msgid "No messages."
+msgstr "[!No messages.!]"
+
+#, python-format
+msgid "%(sender)s wrote:"
+msgstr "[!%(sender)s wrote:!]"
+
+#, python-format
+msgid "%(sender)s sent:"
+msgstr "[!%(sender)s sent:!]"
+
+#, python-format
+msgid "File: %(filename)s"
+msgstr "[!File: %(filename)s!]"
+
+msgid "Download All"
+msgstr "[!Download All!]"
+
+msgid "Delete Source Account"
+msgstr "[!!Delete Source Account!!]"
+
+msgid "Delete All Files and Messages"
+msgstr "[!!Delete All Files and Messages!!]"
+
+msgid "Print Transcript"
+msgstr "[!Print Transcript!]"
+
+msgid "Export Transcript"
+msgstr "[!Export Transcript!]"
+
+msgid "Export All"
+msgstr "[!Export All!]"
+
+msgid "<h2>Some files will not be exported</h2>Some files from this source have not yet been downloaded, and will not be exported.<br /><br />To export the currently-downloaded files, click \"Continue.\""
+msgstr "[!!!!!!!!!!!!!!!!!!!<h2>Some files will not be exported</h2>Some files from this source have not yet been downloaded, and will not be exported.<br /><br />To export the currently-downloaded files, click \"Continue.\"!!!!!!!!!!!!!!!!!!!]"
+
+msgid "all files and transcript"
+msgstr "[!!all files and transcript!!]"
+
+msgid "SecureDrop Client {}"
+msgstr "[!!SecureDrop Client {}!!]"
+
+msgid "Quit"
+msgstr "[Quit]"
```

Comment:
Not to be fixed here, rather in Webate. @cfm What's the workflow for this? Could I do it?

```suggestion
msgid "Quit"
msgstr "[!!Quit!!]"
```

Reply:
Per <https://github.com/freedomofpress/securedrop-client/pull/1548#discussion_r1137937192>, this is the pseduolocale generator working as designed: making both fixed- and variable-length (here, variable length = 0) additions.